### ISSUE TITLE
Create a quiz

### ISSUE THREAD

**Issue by swapagarwal** [swapagarwal/geeksay#64]
We can build a quiz that shows the geeky version and the user needs to type the English translation.
DO NOT PROCEED WITHOUT DISCUSSING IT FIRST!
Let's discuss the scope and implementation details in this issue first. I repeat, please don't create a PR before the details are finalized.

**Comment by wang-ories** [comment]
Hi @swapagarwal ,
Can I help with the quiz?

**Comment by swapagarwal** [comment]
@wang-ories Sure!
Can you first sketch up a quick mock and share the design that's on your mind?

**Comment by wang-ories** [comment]
@swapagarwal Of course, that's sound good!

**Comment by wang-ories** [comment]
<img width="562" alt="quiz" src="https://user-images.githubusercontent.com/52870180/66253402-b8cf1800-e75f-11e9-977f-1cae128f84eb.png">

<img width="564" alt="correct-answer" src="https://user-images.githubusercontent.com/52870180/66253408-c7b5ca80-e75f-11e9-9a1e-ebe21c1c1eeb.png">
<img width="560" alt="incorrect" src="https://user-images.githubusercontent.com/52870180/66253409-c97f8e00-e75f-11e9-836b-9393ec5cc919.png">
<img width="562" alt="result" src="https://user-images.githubusercontent.com/52870180/66253412-cb495180-e75f-11e9-986a-f4343d18ad53.png">

@swapagarwal, here are the design that's in my mind.

**Comment by swapagarwal** [comment]
I think it should be the other way around, i.e. the user should enter the english version as reading the geeky quotes is more fun and typing the original phrase is easier.

**Comment by swapagarwal** [comment]
Also, a "take the quiz" button would need to be added somewhere on the homepage.

**Comment by wang-ories** [comment]
It is also possible to add quiz directly as an option in the home page. i.e : when the user click the button **take quiz**  the section become visible.

**Comment by swapagarwal** [comment]
Agreed, but a dedicated page can be shared more easily on social media.

**Comment by wang-ories** [comment]
I agree too. How about the design of pages ?

**Comment by swapagarwal** [comment]
Looks good. Feel free to go ahead! 💯

**Comment by wang-ories** [comment]
Ok

**Comment by swapagarwal** [comment]
@wang-ories Let me know if you need any help! 😇

**Comment by anmol5varma** [comment]
I can also contribute

**Comment by nativekar** [comment]
Hi @swapagarwal - I have gone ahead and added a section for the quiz as it was not. I have tried and stuck to the brief in having a button on the home screen and rather than handling routing (rather difficult with vanilla js and HTML), I've exercised a modal.
This one is static and will need some handling to make it more dynamic. Happy to help with that, too, if needed.

Please find the PR here: https://github.com/swapagarwal/geeksay/pull/323

**Comment by yamirghofran** [comment]
@wang-ories I'm also open to contribute. Tell me if you want any help or anything else.
